schema/PACK.xsd

```diff
@@ -2095,7 +2095,7 @@
         <!-- Version numbers and release history -->
         <xs:element name="releases" type="ReleasesType" minOccurs="1" />
         <!-- ID based references to changelog information stored in text files, associated with components -->
-        <xs:element name="changeLogs" type="ChangelogsType" minOccurs="0" />
+        <xs:element name="changeLogs" type="ChangeLogsType" minOccurs="0" />
```

Comment:
@jkrech the issue is that `changeLogs` is with a capital L in the .xsd file
the change in the xsd is just a cosmetic change to align with `LicenseSetsType` for `licenseSets`
so if the name is `changeLogs` (with capital L) I expected that the type is `ChangeLogsType` (with capital L) not `ChangelogsType`

Reply:
Wouldn't it be better to change the PACK.xsd then? 
Again my argument was that "changelog" is a single word and I cannot see a reason at this point why we should have a capital "L" in there. What do you think. 
Have you used it much yet?

Reply:
I though changing `changeLogs` to `changelogs` in the xsd is a breaking change
so I avoided doing so, do you prefer that I align the xsd to the documentation instead ?

Reply:
done :)